File: solutions/88-musfik/AgeOfVillagers/Nation/InuitHunters.cs
using System;
using System.Drawing;

namespace AgeOfVillagers
{
    public class InuitHunters : INation
    {
        private readonly Color color = Color.White;
        private String nation_name = "Inuit Hunters";

        public InuitHunters()
        {
        
        }

        public string nationName { get => nation_name; set => nation_name = value; }

        public House getHouse(Point point)
        {
            return new InuitHuntersHouse(point);
        }

       
        public string getNationName()
        {
            return nation_name;
        }
       

        public Color getTerrainColor()
        {
            return color;
        }

        public Tree getTree(Point point)
        {
            return new NoTree(point);
        }

        public WaterResource getWaterResource(Point point)
        {
            return new NoWaterResource(point);
        }
    }
}

File: solutions/72-rafidhaque/src/InuitHunters.java
import javafx.scene.Group;
import javafx.scene.paint.Color;
import javafx.scene.shape.Rectangle;

public class InuitHunters implements INation {
    String nationName = "Inuit Hunters";
    Group drawSpace;
    DrawHouse house;
    DrawTree tree;
    DrawWater water;

    public InuitHunters(Group drawSpace) {
        this.drawSpace = drawSpace;
        Rectangle fillColor = new Rectangle(0,0, 600, 400);
        fillColor.setFill(Color.WHITE);
        drawSpace.getChildren().add(fillColor);
        this.house = new InuitHuntersHouse();
        this.tree = new NoTree();
        this.water = new NoWater( 0, 0);
        drawSpace.getChildren().addAll(house.draw());
        drawSpace.getChildren().addAll(tree.draw());
        drawSpace.getChildren().addAll(water.draw());
    }

    public String getNationName() {
        return nationName;
    }

    public DrawHouse getHouse() {
        return house;
    }

    public DrawTree getTree() {
        return tree;
    }

    public DrawWater getWater() {
        return water;
    }
}

